playbooks/roles/edxapp/templates/edx/app/edxapp/lms.sh.j2

```diff
@@ -17,7 +17,18 @@ if command -v ec2metadata >/dev/null 2>&1; then
   HOSTNAME=$(hostname)
   export NEW_RELIC_PROCESS_HOST_DISPLAY_NAME="$HOSTNAME-$INSTANCEID"
 fi
+
+
+{% if COMMON_DEPLOYMENT == "edge" %}
```

Comment:
Wouldn't we want this for Stage and Edge, to test in Stage first?

Reply:
True. Even if I don't think it will help us debug in stage, we should still double-check that it doesn't break there. (I did local testing in devstack, but something could be different.)